src/__tests__/is-ipados.ts

```diff
@@ -0,0 +1,45 @@
+import isIpadOS = require("../is-ipados");
+
+const AGENTS: {
+  [key: string]: string;
+} = require("./helpers/user-agents.json");
+
+const DOCUMENT_OBJECT = {
+  ontouchend: null,
+};
+
+const DOCUMENT_OBJECT_EMPTY = {};
+
+describe("isIpadOS", () => {
+  it("returns true for an iPad", () => {
```

Comment:
what's the difference between this test and the test on line 21?

Reply:
ope, sorry looks like a typo on my side, I meant the test with the same name on line 28, what's the difference between these two tests?

Reply:
oh! that one is for cases where iPad is lowercase `ipad`, I just copied that out of the ios test from where it was originally